engines/scumm/script_v6.cpp

```diff
@@ -555,6 +628,105 @@ void ScummEngine_v6::o6_eq() {
 	int a = pop();
 	int b = pop();
 
+// BACKYARD BASEBALL 2001 ONLINE CHANGES
+#if defined(USE_ENET) && defined(USE_LIBCURL)
+	if (ConfMan.getBool("enable_competitive_mods")) {
+			// People have been complaining about strikes being visually unclear during online games. This is because the strike zone's visual is not
+			// equal length compared to the actual range in which a strike can be called. These changes should fix that, with some extra leniency in
+			// the corners in particular since they are especially difficult to see visually, due to having four large corner pieces blocking the view.
+
+			// This checks if the pitch's y location is either:
+			// a. at least 2 pixels lower than the top of the zone/at least 3 pixels above the bottom of the zone
+			// b. at least 2 pixels lower than the top of the zone/at least 3 pixels above the bottom of the zone
+			// If either of these are true AND the x value is less than or equal to 279 OR greater than or equal to 354, make the game read as a ball.
+			// The strike zone should be much more lenient in the corners, as well as removing the small advantage of throwing to the farthest right side of the zone.
+			if (_game.id == GID_BASEBALL2001 && _currentRoom == 4 && (vm.slot[_currentScript].number == 2202 || vm.slot[_currentScript].number == 2192) && readVar(399) == 1) {
+				if (((readVar(0x8000 + 12) <= readVar(0x8000 + 29) + 2 || readVar(0x8000 + 12) >= readVar(0x8000 + 30) - 3) && readVar(0x8000 + 11) <= 279) ||
```

Comment:
`readVar(0x8000 + 12)` is used 5 times in this block. Consider assigning it to a variable, to improve readability

Reply:
I would leave it as such, since I would rather create a macro myself down the line, like:
- `ROOM_VAR(x)` for `readVar(0x8000 + x)` 
- `LOCAL_VAR(x)` for `readVar(0x4000 + x)`

Reply:
So is there much more that I need to do?

Reply:
Thanks! Much better now :)